documentation/getting-started/getting-started-kotlin.asciidoc

```diff
@@ -0,0 +1,164 @@
+---
+title: Using Vaadin with Kotlin
+order: 600
+layout: page
+---
+
+= Using Vaadin with Kotlin
+
+Vaadin supports the Kotlin programming language through the Karibu-DSL library, which contains various extensions to use Vaadin with Kotlin.
+Perhaps most importantly, the library allows you to create your Vaadin UI in a declarative way using a _domain-specific language_ (DSL).
+These features are enabled by the http://www.vaadinonkotlin.eu/index.html[Vaadin on Kotlin] project.
```

Comment:
Vaadin-on-Kotlin is the Karibu-DSL + custom database support + other things; I'd only focus on the DSL part and link only the Karibu-DSL from here:

https://github.com/mvysny/karibu-dsl[Karibu-DSL]